src/odoc/page-index.mld

```diff
@@ -0,0 +1,252 @@
+{1:top odoc}
+
+The documentation compiler for OCaml, Reason, and BuckleScript.
+
++ {{:#overview} What is [odoc] really? }
++ {{:#flow} How does it work? }
++ {{:#deep-dive} Digging Deeper }
+
+{2:overview What is [odoc] really?}
+
+[odoc] is not your regular documentation tool.
```

Comment:
I'd mention that it's designed to be build-tool friendly. With focus on parallelism, caching, etc. I don't mind adding those changes separately though.

Reply:
Rather than "regular" I'd say "ordinary". We hope it becomes everyone's _regular_ tool. :)